inventory/routes/concepts/pull_push_rules.rst

```diff
@@ -0,0 +1,101 @@
+
+Configuration
+=============
+
+| Pull and push rules are part of the routes. Go to the *Inventory
+  application, :menuselection:`Inventory --> Configuration --> Settings` and tick *Multi-Step Routes*.
+
+.. image:: media/pull_push_rules_01.png
+   :align: center
+   :alt: Enabling multi-step routes to use pull and push rules
+
+.. Note::
+   Storage Locations are automatically activated at the same time as
+   Multi-Step Routes.
+
+Rules settings
+==============
+
+| The rules are defined on the routes. In the inventory application, go to
+  :menuselection:`Inventory --> Configuration --> Routes`. In the *Rules* section, click on *Add a line*.
+
+.. image:: media/pull_push_rules_02.png
+   :align: center
+   :alt: Add a line on Odoo routes to use Pull & Push rules
+
+| Here you can set the action to be triggered by the rule. Choose from:
+
+-  **Pull From**: this rule will be triggered by a need for the product in a specific stock location. The need can come from a sale order which has been validated or for a manufacturing order which requires a specific component. When the need appears in the source location, Odoo generates a picking to fulfill this need thanks to the pull rule.
+-  **Push To**: this rule will be triggered by the arrival of some products in the defined source location. In case you move products to the source location, Odoo generates a picking to move those products to the destination location.
+-  **Push & Pull**: it allows to generate a picking in the two different situations explained above, when products are required in a specific source location as well as when they are pushed to a specific source location.
+-  **Buy**: when products are needed the source location, a request for quotation is created to fulfill the need.
+-  **Manufacture**: when products are needed in the source location, a manufacturing order is created to fulfill the need.
+-  Many translated example sentences containing
+
+.. image:: media/pull_push_rules_03.png
+   :align: center
+   :alt: Creating rules inside of Routes
+
+| The *Operation Type* allows you to define which kind of picking will
+  be created from this rule, in the example, I would like to create an
+  internal transfer.
+| The *Supply Method* allows you to define what happens at the source
+  location:
+
+-  **Take From Stock**: the products will be taken from the available stock of the source location.
+-  **Trigger Another Rule**: the system will try to find a stock rule to bring the products in the source location. The available stock will be ignored.
+-  **Take From Stock, if Unavailable, Trigger Another Rule**: the products will be taken from the available stock of the source location. If there is no stock available, the system will try to find a rule to bring the products in the source location.
+
+| In the *Scheduling* section, you can determine how will Odoo behave
+  when one of the pickings of the chain is rescheduled. In case you decide
```

Comment:
```suggestion
In the *Scheduling* section, you can determine how Odoo behaves
when one of the pickings of the chain is rescheduled. In case you decide
```